### PR TITLE
Major Improvements

### PR DESCRIPTION
**Pull request by axelstudios**
## Improvements:

- Angular 14 upgrade
- Localized fonts
- Upgraded dependencies
- Major responsive layout improvements
- Updated all links that redirect to a different URL
- Fixed broken links
- Fixed a bug where the Asset Score Goals page incorrectly linked to the old GitHub.io version of the site instead of the local page
- Fixed mixed tabs/spaces
- Fixed inconsistent indentation
- Fixed typos
- Fixed lint errors
- Optimized images
- Removed unused files
- Added quick fade to home card hover
- Fixed nav hover so that empty space on the sides doesn't expand it
- Nav now overlays on hover instead of causing it to jump
- Fixed homepage padding
- Improved SEED allies layout
- Improved BuildingSync collaborators layout
- Fixed image brightness on homepage when opaque
- Cybersecurity improvements: automatic `target="_blank" rel="noopener"` on all external links
- Automatic external link icons
- Replaced deprecated code
- Fixed ngx-markdown sanitization warnings
- Updated BETTER news
- Updated readme
- Added buildingsync.net link to BuildingSync page
- Reviewed and fixed links in markdown files
- Fixed horizontal scrollbar from EERE footer
- Improved data standardization responsive layout

### CODE REVIEW

**Review by kflemin — APPROVED** [review]
🏆 looking good!!! If i were super picky, I'd say the 'data standardization' page graphic could look better on smaller resolutions (ipad mini), and the whole site doesn't quite fit at ipads / surface pro resolution (there's some horizontal scrolling)...otherwise good stuff!

File: README.md [hunk 1]
```diff
@@ -37,14 +37,14 @@ to this lab's branch to this repository (pnnl, lbnl, nrel). The sites are locate
...
 
 * Mac OSX: `brew install npm`
 * Linux: `apt install npm`
 * Windows: TBD
 
 After npm is installed, go into the root directory from a clone of this repository and run the following (note that you may need to
-run `npm install -g @angular/cli` to have the Angular CLI `ng` be in the user's path). 
+run `npm install -g @angular/cli` to have the Angular CLI `ng` be in the user's path).
 
 *Note that we are pinning to an older version of Node (12.22.5)*
```
[the remark this answers is not in the document]
Comment (axelstudios) [reply README.md:49]: v16.10 or newer is ideal, v12-v14 are pretty old at this point

[the remark this answers is not in the document]
Comment (axelstudios) [reply README.md:49]: I saw the horizontal scrolling, too - I'm pretty sure that comes from the badly designed EERE footer

[the remark this answers is not in the document]
Comment (kflemin) [reply README.md:49]: v12 didn't work at all, so we should update the readme with a newer version

[the remark this answers is not in the document]
Comment (nllong) [reply README.md:49]: I just pushed an update in the README to use 16.18.0 which is what the CI is using.

**Review by nllong — APPROVED** [review]
This looked great. Site ran as expected with the updated "drop down" that doesn't move all the text.

**Pull request merged**